pysma/__init__.py

```diff
@@ -142,6 +143,10 @@ async def _request_json(
                 raise SmaConnectionException(
                     f"Could not connect to SMA at {self._url}: {exc}"
                 ) from exc
+            except asyncio.TimeoutError:
+                raise SmaConnectionException(  # pylint: disable=raise-missing-from
```

Comment:
Why disable the pylint warning instead of using `from exc`?

Reply:
Raising from often gives a very lengthy stack trace. Here we know the timeout is from somewhere deep in the aiohttp client library, but we really don't care exactly where it comes from.

But a personal (maybe a bit opinionated) preference and we could use either!

Reply:
As the raised exceptions above this new one use `from exc` it might be best to keep this one in the same format?